### ISSUE TITLE
Colorful Tic Tac Toe Game with GUI using tkinter

### ISSUE THREAD

**Issue by Sanskriti1102** [The-Data-Alchemists-Manipal/MindWave#557]
### Is your feature request related to a problem? Please describe.

Yes, the feature request is related to enhancing the existing Tic Tac Toe Python file. The current implementation only displays the board in the command line when the file is run. 

### Describe the solution you'd like

I would like to add some features using Tkinter to enhance the Tic Tac Toe game. The desired solution is to create a graphical interface where a board will be displayed, and players can interact with the game by clicking on the buttons representing the positions on the board. The interface should allow players to make their moves and check for a winner based on the updated board.

### Describe alternatives you've considered

I have also considered the possibility of displaying player statistics as an alternative feature. This would involve keeping track of each player's win count, loss count, and tie count throughout multiple game sessions. By displaying these statistics, players would be able to track their performance and evaluate their gameplay over time.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Sanskriti1102** [comment]
Could you please assign me this issue under GSSOC'23?

**Comment by khusheekapoor** [comment]
@Sanskriti1102 -  you can go ahead! We are assigning you 21 days for this project, after which it will be assigned to someone else if not completed. All the best! 
Name the file as: `algorithm_dataset.ipynb` and link it in the readme of the labeled directory as `algorithm - dataset`.

**Comment by Sanskriti1102** [comment]
Yes, Ma'am will do it within 21 days. Thank You @khusheekapoor Ma'am for assigning me this issue

**Comment by Sanskriti1102** [comment]
@khusheekapoor Ma'am, I have named the file as algorithm_dataset,ipynb and wanted to link it in the readme so can you please guide me regarding the above?

**Comment by Sanskriti1102** [comment]
@khusheekapoor  Ma'am, i have completed the issue and have done as per what you have told

this is my code for the above :
https://github.com/Sanskriti1102/MindWave/blob/main/Machine%20Learning/Obesity%20Classification/Algorithm-dataset/algorithm_dataset.ipynb

This is the overview for this code:
https://github.com/Sanskriti1102/MindWave/blob/main/Machine%20Learning/Obesity%20Classification/Algorithm-dataset/algorithm_dataset
